### DISCUSSION TITLE
Release notifications

### DISCUSSION THREAD

**Discussion by Awlexegrecki** [community/community#139237] [Mobile]
I believe a needed feature for the Github app would be an addition to the notification section that allowed the user to receive notifications when a new release of a starred or favorited repository rolls out.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.178.0 (141531614)
Device: iPhone 15 Pro (iOS 18.1)
</details>

**Comment by kangarruu** [comment]
@Awlexegrecki Thank you for your suggestion! This is a great idea and something we've actively explored. While there are no immediate plans to implement it at this time, we hope to revisit it in the future. We appreciate your input and hope you keep writing in with similar feature requests!
